### ISSUE TITLE
Callout descriptions are always autonumbered ignoring the explicit numbers

### ISSUE THREAD

**Issue by TheDet** [asciidoctor/asciidoctor#4105]
Using explicit callout numbers in the code as well as in the description leads to discrepancies due to the description always being autonumbered:

[source,java]
----
public void test() {
      lineOne();  // <2>
      lineTwo();  // <3>
}
----
<2> This is shown as 1.
<3> This is shown as 2.

Attached you find an adoc file showing the source, and a PDF demonstrating the result.
(Note: This is not a PDF converter's problem, as the same is seen in HTML conversion and the IntelliJ Plugin's preview).

This issue was seen and the files produced in Asciidoctor IntelliJ Plugin 0.33.6 .

NB: It is not possible to upload a file with the .adoc extension, so I had to add the postfix .txt to it.

[Callout-Autonumbering-Problem.pdf](https://github.com/asciidoctor/asciidoctor/files/6868095/Callout-Autonumbering-Problem.pdf)
[Callout-Autonumbering-Problem.adoc.txt](https://github.com/asciidoctor/asciidoctor/files/6868099/Callout-Autonumbering-Problem.adoc.txt)

**Comment by mojavelinux** [comment]
You are correct. This is by design. The numbers either have to start with 1, or you can use autonumbers. And you shouldn't mix them.
